flang/lib/Lower/ConvertVariable.cpp

```diff
@@ -1345,8 +1356,9 @@ void Fortran::lower::mapSymbolAttributes(
         if (x.lboundAllOnes()) {
           // if lower bounds are all ones, build simple shaped object
           llvm::SmallVector<mlir::Value> shape;
-          for (auto i : x.shapes)
-            shape.push_back(builder.createIntegerConstant(loc, idxTy, i));
+          for (auto i : x.shapes) {
+            shape.push_back(genExtentValue(builder, loc, idxTy, i));
+          }
```

Comment:
Did you mean to reformat this `for `statement to use a brace-enclosed statement?